scripts/simp_golfing.py

```diff
@@ -0,0 +1,123 @@
+
+
+import os
+import re
+
+def extract_simp_rw_from_simp_only(filename):
+
+    lines = list(open(filename, "r"))
+
+    # Find first line with a simp_only
+    for line in list(lines):
+        if "simp only" in line:
+            break
+
+    print(f"Bot found line {line}")
+
+    simp_only_start = line.find("simp only")
```

Comment:
This also matches `dsimp only`

Reply:
Yes, it should probably be replaced by a regex, I would have to learn how to write those.